src/Nethermind/Nethermind.JsonRpc/Modules/Eth/SimulateTxExecutor.cs

```diff
@@ -120,91 +121,67 @@ public override ResultWrapper<IReadOnlyList<SimulateBlockResult>> Execute(
                 $"Too many blocks provided, node is configured to simulate up to {_blocksLimit} while {call.BlockStateCalls?.Count} were given",
                 ErrorCodes.InvalidParams);
 
-        secondsPerSlot ??= new BlocksConfig().SecondsPerSlot;
+        ulong secondsPerSlot = new BlocksConfig().SecondsPerSlot;
```

Comment:
you cannot rely on `new` here, as you will ignore configured value

Reply:
Okay, this is what we had before, let me change it.